lighthouse-cli/chrome-launcher.ts

```diff
@@ -233,12 +234,18 @@ class ChromeLauncher {
 
     log.verbose('ChromeLauncher', `Removing ${this.TMP_PROFILE_DIR}`);
 
-    if (this.outFile) {
-      fs.closeSync(this.outFile);
-    }
+    try {
```

Comment:
I'd rather avoid the try/catch since it'll silently eat errors, but is it even necessary? This is the only place that these files are closed, so the conditionals should guard against attempting to delete with already-closed file descriptor.